### PR TITLE
[Fluent] Add tabstrip styles

### PR DESCRIPTION
**Pull request by epetrow**
Related to: https://github.com/telerik/kendo-themes-private/issues/3

### CODE REVIEW

File: packages/fluent/scss/tabstrip/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+/// @group tabstrip
+$kendo-tabstrip-item-dragging-text: get-theme-color-var( primary-130 ) !default;
+
+// Indicator
+$kendo-tabstrip-indicator-size: null !default;
+$kendo-tabstrip-indicator-color: null !default;
+
+/// The horizontal padding of tabstrip content
+/// @group tabstrip
+$kendo-tabstrip-content-padding-x: map-get( $kendo-spacing, 2 ) !default;
```
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_variables.scss:119]: According to the design this should be 12px (`map-get( $kendo-spacing, 2 )`)

File: packages/fluent/scss/tabstrip/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+/// The vertical padding of tabstrip content
+/// @group tabstrip
+$kendo-tabstrip-content-padding-y: map-get( $kendo-spacing, 3 ) !default;
```
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_variables.scss:122]: According to the design this should be 6px (`map-get( $kendo-spacing, 1 ) + map-get( $kendo-spacing, thin )`)

File: packages/fluent/scss/tabstrip/_layout.scss [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+        border-width: 0;
+        border-style: solid;
+        border-color: inherit;
+        display: none;
+        overflow: auto;
+        flex: 1 1 auto;
+        position: relative;
+        z-index: 1;
+
+        &.k-state-active,
```
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_layout.scss:118]: Delete the `&.k-state-active` selector.

File: packages/fluent/scss/tabstrip/_layout.scss [hunk 4]
```diff
@@ -0,0 +1,429 @@
...
+                    border-top-width: var( --kendo-tabstrip-indicator-size, #{$kendo-tabstrip-indicator-size} );
+                }
+                .k-tabstrip-left > & {
+                    border-right-width: var( --kendo-tabstrip-indicator-size, #{$kendo-tabstrip-indicator-size} );
+                }
+                .k-tabstrip-right > & {
+                    border-left-width: var( --kendo-tabstrip-indicator-size, #{$kendo-tabstrip-indicator-size} );
+                }
+            }
+            .k-item.k-state-active::after,
```
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_layout.scss:397]: Delete the `.k-item.k-state-active::after` selector.

File: packages/fluent/scss/tabstrip/_theme.scss [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+        &:focus,
+        &.k-focus {
+            outline-color: var( --kendo-tabstrip-content-border-focused, #{$kendo-tabstrip-content-border-focused} );
+        }
+    }
+
+
+    // Selected indicator
+    @if ($kendo-tabstrip-indicator-size) {
+
```
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_theme.scss:98]: Either remove this empty line or add an empty line after line 103 😊

File: packages/fluent/scss/tabstrip/_variables.scss [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+/// The background color of the disabled tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-disabled-bg: none !default;
+/// The border color of the disabled tabstrip item
+/// @group tabstrip
+$kendo-tabstrip-item-disabled-border: var( --kendo-disabled-border, initial ) !default;
+
+/// The background color of hovered tabs
+/// @group tabstrip
+$kendo-tabstrip-item-hovered-bg: get-theme-color-var( neutral-20 ) !default;
```
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_variables.scss:87]: Use the CSS variable (`var( --kendo-hover-bg, inherit )`)

File: packages/fluent/scss/tabstrip/_variables.scss [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+/// The text color of hovered tabs
+/// @group tabstrip
+$kendo-tabstrip-item-hovered-text: get-theme-color-var( neutral-190 ) !default;
```
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_variables.scss:90]: Use the CSS variable (`var( --kendo-hover-text, inherit )`)

File: packages/fluent/scss/tabstrip/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+/// The border color of hovered tabs
+/// @group tabstrip
+$kendo-tabstrip-item-hovered-border: get-theme-color-var( neutral-20 ) !default;
```
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_variables.scss:93]: Use the CSS variable (`var( --kendo-hover-bg, inherit )`)

File: packages/fluent/scss/tabstrip/_variables.scss (hunk 2 above)
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_variables.scss:122]: According to the design this should be 6px (`map-get( $kendo-spacing, 1 ) + map-get( $kendo-spacing, thin )`)

File: packages/fluent/scss/tabstrip/_variables.scss (hunk 1 above)
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_variables.scss:119]: According to the design this should be 12px (`map-get( $kendo-spacing, 3 )`)

File: packages/fluent/scss/tabstrip/_variables.scss (hunk 7 above)
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_variables.scss:90]: Use the CSS variable (`var( --kendo-hover-text, inherit )`)

File: packages/fluent/scss/tabstrip/_variables.scss (hunk 6 above)
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_variables.scss:87]: Use the CSS variable (`var( --kendo-hover-bg, inherit )`)

File: packages/fluent/scss/tabstrip/_theme.scss (hunk 5 above)
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_theme.scss:98]: Either remove this empty line or add an empty line after line 103 😊

File: packages/fluent/scss/tabstrip/_layout.scss (hunk 4 above)
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_layout.scss:397]: Delete the `.k-item.k-state-active::after` selector.

File: packages/fluent/scss/tabstrip/_layout.scss (hunk 3 above)
Comment (JoomFX) [inline packages/fluent/scss/tabstrip/_layout.scss:118]: Delete the `&.k-state-active` selector.

**Review by JoomFX — CHANGES_REQUESTED** [review]

**Review by Juveniel — APPROVED** [review]

**Pull request merged**